service/lib/agama/dbus/manager_service.rb

```diff
@@ -70,7 +72,12 @@ def start
 
       # Exports the installer object through the D-Bus service
       def export
+        # users service initialization
+        users_service = bus.request_service("org.opensuse.Agama.Users1")
+
+        # manager service initialization
         dbus_objects.each { |o| service.export(o) }
+        users_service.export(Agama::DBus::Users.new(Agama::Users.new(logger), logger))
```

Comment:
```suggestion
        service.export(Agama::DBus::Users.new(Agama::Users.new(logger), logger))
```

when I do this (which is an obscure workaround!)  it starts to work:

```console
ba3c5497c2c2:/ # abusctl tree org.opensuse.Agama.Users1
Only root object discovered.
```

Oops. Ruby-dbus has a race condition, exposing the service before you export any objects. Let's try again.

```console
ba3c5497c2c2:/ # abusctl tree org.opensuse.Agama.Users1
└─/org
  └─/org/opensuse
    ├─/org/opensuse/Agama
    │ └─/org/opensuse/Agama/Users1
    └─/org/opensuse/Agama1
      └─/org/opensuse/Agama1/Manager
```